manuscript/draft.Rmd

```diff
@@ -259,59 +260,63 @@ on these time scales once observer differences are accounted for.
 When comparing forecasts for richness across sites all methods performed well
 (Figure 2; R^2 XX-XX%). However stacked and joint SDMs and the macroecological
 model both failed to successfully forecast the highest richness sites resulting
-in a noteable clustering of predicted values near ~60 species and the poorest
+in a notable clustering of predicted values near ~60 species and the poorest
 performance in the cross-site comparison (R^2 XX-XX%).
 
 For most combinations of metric and forecast time lag, the best-performing model
-for forecasting within site variability was either the "average" baseline model
-or the ARIMA model (Figure 3). In general, each model's error increased
-more-or-less linearly with time, although the slopes and initial error rates
-varied across models (Figure 3). The site-level models (i.e. the baselines and
-the ARIMA methods) tended to do very well (Figure 3) on short time horizons,
-especially in terms of RMSE.
+for forecasting within-site variability was either the "average" baseline model
+or the ARIMA model [[Not sure what the bit about within-site variability means
+here. Also, didn't Naive usually win when observer effects weren't corrected
+for?]] (Figure 3). In general, each model's error increased more-or-less
+linearly with time, although the slopes and initial error rates varied across
+models (Figure 3). The site-level models (i.e. the baselines and the ARIMA
+methods) tended to do very well (Figure 3) on short time horizons, especially in
+terms of RMSE.
 
 In general, the models produced confidence intervals that were too narrow,
 indicating overconfident predictions. Of these, the random forest-based SDM
-stack was the most overconfident. As noted elsewhere [@harris2015], these models
-are generally far too confident, with observed richness values frequently
-falling well outside their confidence limits (Figure 3C). The "naive" model was
-the only model whose confidence intervals were too wide (Figure 3), which can be
-attributed to the rapid rate at which these intervals expand as the time lag
-increases (Figure 1). The JSDM and macroecological model has the best coverage.
+stack was the most overconfident, with observed richness values frequently
+falling well outside their confidence limits (Figure 3C). As noted elsewhere
+[@harris2015], this overconfidence is a product of the assumption in stacked
+SDMs that errors in the species-level predictions are independent. The JSDM and
+macroecological model has the best coverage, though each model still
```

Comment:
"has" -> "had" (while acknowledging that it was my typo that you just copied and pasted.)